Sources/Knit/Module/ModuleAssembly.swift

```diff
@@ -40,6 +45,24 @@ public extension ModuleAssembly {
             return self.resolverType == $0.resolverType
         }
     }
+
+    static var _assemblyFlags: [ModuleAssemblyFlags] {
```

Comment:
```suggestion
    static let _assemblyFlags: [ModuleAssemblyFlags] = {
```
Can we make this a computed static property so it is only calculated once per assembly type?

Reply:
I can't add a static let in an extension